testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1384,7 +1384,9 @@
   success = file_inject(get_target('server'), source, dest)
   raise ScriptError, 'File injection in server failed' unless success
 
-  get_target('server').run("expect -f /tmp/#{file} #{$reportdb_ro_user}")
+  node = get_target('server')
+  mgrctl = node.has_mgrctl ? 'mgrctl' : ''
```

Comment:
It was a bit hard to understand the first time that I read it :)
But ok

Reply:
Just maybe, we can reduce one line by directly using true/false values coming from `node.has_mgrctl` instead of mapping those to 'mgrctl'/''.
But a matter of taste.

Reply:
I'm not an expert in `expect` syntax. I can try your suggestion, I have no clue if that will be properly interpreted.